pkg/compactor/block_upload_test.go

```diff
@@ -109,6 +109,7 @@ func TestMultitenantCompactor_HandleBlockUpload_Create(t *testing.T) {
 		body                   string
 		meta                   *metadata.Meta
 		retention              time.Duration
+		enableBlockUpload      bool
```

Comment:
I know it's not great practice, but if we called this `disableBlockUpload`, we don't need to modify every single test.

Reply:
Thanks, done.